CHANGELOG.md

```diff
@@ -59,6 +59,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Refactor `AgentManager::ping` to return `Duration` instead of `i64`. This
   refactor improves the flexibility and accuracy of the `ping` method, making it
   more robust and aligned with Rust's time handling conventions.
+- Modified the functions that inserts and updates the account to include the language
```

Comment:
Could you check this sentence's grammar with external translation tools please?

Reply:
Sorry, I've corrected the wrong part.